src/tests/test_latest.py

```diff
@@ -2,11 +2,9 @@
 
 
 def test_is_latest():
-    last_millesime_name = ""
-    for member in Millesime:
-        if member.name > last_millesime_name:
-            last_millesime_name = member.name
+    recent_millesime_name = max((member.name for member in Millesime))
+    recent_millesime = Millesime[recent_millesime_name]
 
-    last_millesime = Millesime[last_millesime_name]
-
-    assert Millesime.LATEST == last_millesime, "LATEST is not the most recent Millesime"
+    assert (
+        Millesime.LATEST == recent_millesime
+    ), "LATEST is not the most recent Millesime"
```

Comment:
```suggestion
    ), "LATEST is not the most recent available Millesime"
```

Suggestion, juste pour qu'il n'y ait pas d'ambiguité sur le fait que les données peuvent ne pas être à jour, et le test passer malgré tout :)